share/ci/docker/ubuntu18.04_vfx2019/Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM ubuntu:18.04
+
+COPY share/ci/scripts/* /tmp/
+
+RUN /tmp/linux/apt/install.sh && \
+    /tmp/linux/apt/install_gcc.sh 6 && \
+    /tmp/linux/apt/install_clang.sh 6.0 && \
+    /tmp/linux/apt/install_python.sh 2.7 && \
+    /tmp/linux/install_numpy.sh 1.14 && \
+    /tmp/linux/install_cmake.sh 3.14.4 && \
+    /tmp/linux/install_truelight.sh 4.0.7863 && \
+    /tmp/linux/install_boost.sh 1.66.0 && \
+    /tmp/linux/install_openexr.sh 2.3.0 && \
+    /tmp/linux/install_oiio.sh 2.0.8 && \
```

Comment:
FYI, OIIO has a "release" branch marker that is kept up to whatever is the latest official production release. Using that instead of a hard-coded version might save you the trouble of having to bump this number every month or so that we do OIIO patch releases.